src/main/java/com.marketdata.api/MarketDataController.java

```diff
@@ -17,6 +15,6 @@ public String index() {
 
     @GetMapping("/MarketData/{symbol}")
     public String fetchSymbol(@PathVariable("symbol") String symbol) {
-        return "index";
+        return symbol;
```

Comment:
Hmmm.   This should return the name of an HTML page.  "index" feels appropriate here; not the symbol parameter passed in.  I'd leave this as it were.